oauth2client/client.py

```diff
@@ -194,15 +194,16 @@ class Credentials(object):
     NON_SERIALIZED_MEMBERS = frozenset(['store'])
 
     def authorize(self, http):
-        """Take an httplib2.Http instance (or equivalent) and authorizes it.
+        """Creates an http object that provides credentials to requests.
 
-        Authorizes it for the set of credentials, usually by replacing
-        http.request() with a method that adds in the appropriate headers and
-        then delegates to the original Http.request() method.
+        Typically will use :mod:`transport` to wrap the http object to provide
+        credentials to requests and handle refreshing credentials as needed.
 
         Args:
-            http: httplib2.Http, an http object to be used to make the refresh
-                  request.
+            http: The http object to wrap.
```

Comment:
Is this type ambiguous? Is the type of this parameter one that we've defined, one that exists among the community of libraries of which this library is a part, or is made-up and hand-wavy?